subsys/bluetooth/host/Kconfig

```diff
@@ -703,7 +703,8 @@ config BT_SMP_MIN_ENC_KEY_SIZE
 	default 16 if BT_SMP_SC_ONLY
 	default 7
 	help
-	  This option sets the minimum encryption key size accepted during pairing.
+	  This option sets the minimum encryption key size accepted during pairing
+	  both for LE and Classic.
```

Comment:
Before this PR, the kconfig is only used by LE. So I updated it (Refer the comments from other reviewer).

Reply:
> Before this PR, the kconfig is only used by LE. So I updated it (Refer the comments from other reviewer).

Sure, but most of the Kconfigs that apply to both LE and BR/EDR does not state this :) For example `CONFIG_BT_MAX_CONN`

Reply:
> Sure, but most of the Kconfigs that apply to both LE and BR/EDR does not state this :) For example `CONFIG_BT_MAX_CONN`

Right, but `BT_MAX_CONN` doesn't have an LE-specific word in it. `SMP` is an LE-specific protocol, so it may not be clear that it'll get used for BR/EDR SSP as well. That said, I think it'd be better to remove "SMP" from the option name completely, or if we want to avoid API breaks have a shadow option which has the same value.

Reply:
Ah. 

So yeah, we can remove SMP from the option (BR/EDR seems to have quite a few things in the SMP implementation though...), or we can introduce another BR/EDR specific Kconfig option. 

Is there any use case where you'd want to have e.g. minimum 12 for BR/EDR and 16 for LE?

Reply:
I haven't encountered this case yet.